### DISCUSSION TITLE
PP-OCRv4训练出的det模型多很多reshape

### DISCUSSION THREAD

**Discussion by uniyushu** [PaddlePaddle/PaddleOCR#12318] [Q&A]
请提供下述完整信息以便快速定位问题/Please provide the following information to quickly locate the problem

- 系统环境/System Environment：
<img width="539" alt="image" src="https://github.com/PaddlePaddle/PaddleOCR/assets/20262193/8daf45a5-cab5-4d99-8dc4-6f92c3b3c0d5">

- 版本号/Version：Paddle：  PaddleOCR

<img width="537" alt="image" src="https://github.com/PaddlePaddle/PaddleOCR/assets/20262193/c2f9bb4b-fd19-4592-991d-a9bfae2693fa">

- 运行指令/Command Code：
```
python tools/train.py -c /root/ysc/motor_e2e/test_img/ch_PP-OCRv4_det_student.yml
python tools/export_model.py -c /root/ysc/motor_e2e/test_img/ch_PP-OCRv4_det_student.yml -o Global.pretrained_model="/root/ysc/motor_e2e/output/ch_PP-OCRv4/best_model/model.pdparams" Global.save_inference_dir="/root/ysc/motor_e2e/output/det_v4_student_inference/"
```
- 完整报错/Complete Error Message：

我利用[ch_PP-OCRv4_det_student.yml](https://github.com/PaddlePaddle/PaddleOCR/blob/release/2.7/configs/det/ch_PP-OCRv4/ch_PP-OCRv4_det_student.yml)训练出的模型和[paddleOCRV4 inference 模型](https://paddleocr.bj.bcebos.com/PP-OCRv4/chinese/ch_PP-OCRv4_det_infer.tar) 多出了很多reshape，请问怎么去掉这些reshape使训练模型之后导出的模型和官方给的模型一致呢？
<img width="419" alt="企业微信截图_ae6612db-e956-4753-9099-682fd3b06418" src="https://github.com/PaddlePaddle/PaddleOCR/assets/20262193/ee04715b-cbd8-43ed-bbf8-a1d470743ceb">
<img width="378" alt="企业微信截图_6fa57e53-d536-49b0-acd2-a488317f2209" src="https://github.com/PaddlePaddle/PaddleOCR/assets/20262193/2657717c-095e-4cbf-9642-18bdf4a07394">
<img width="391" alt="image" src="https://github.com/PaddlePaddle/PaddleOCR/assets/20262193/3fc6971d-1ef6-4af9-a63c-0e1581fa910d">
<img width="438" alt="image" src="https://github.com/PaddlePaddle/PaddleOCR/assets/20262193/570ceb17-39a3-4fbd-afb6-b1864f67cc61">

**Comment by ysong-aceinna** [comment]
会不会是因为预训练模型和.yml不匹配导致的，如果用ch_PP-OCRv4_det_cml.yml有这个问题吗？
